Fix capitalization and links on Feature opt-in page (#967)
DevCycleHQ/devcycle-docs: docs/platform/extras/feature-opt-in.mdx

If you have an idea for this widget but don't know how to execute it, just reach out to support@devcycle.com and we'd be happy to help you figure out the best path forward.

https://youtu.be/jqAY-3Y4T7k

# Implementation

Feature Opt-In allows end users of your product to opt-in to a list of eligible Features through a pre-built widget that can be implemented in your app and website with one line of code. Your team has the ability to customize the look & feel of this widget to your company's colour profile and logo. 

This article explains how to implement our Feature Opt-In functionality on your app or website. 

## Requirements

To get started with the Feature Opt-In functionality all you need are running Features in DevCycle and a page that you can install the iFrame into.

## Getting Started

The Feature Opt-In functionality can be turned on from the [Project Settings](https://app.devcycle.com/r/optIn) page.

Once in your Project Settings, navigate to the "Feature Opt-In" page:

![Feature Opt-In Settings](/oct-22-2022-feature-opt-in-1.png)

> Feature Opt-In is first enabled at the Project level. It needs to be enabled manually because it leverages EdgeDB to manage a user's opt-in state. As a result, Feature Opt-In is not available on Free plans at this time. Visit our [pricing page](https://devcycle.com/pricing) for more details.

Author: ```suggestion
Feature Opt-In is first enabled at the Project-level. It needs to be enabled manually because it leverages EdgeDB to manage a user's opt-in state. This means that enabling Feature Opt-In on a project can impact billing for your account. For more information on EdgeDB pricing visit our [pricing page](https://devcycle.com/pricing).
```

To enable Feature Opt-In, switch the toggle to "Opt-In Enabled".

![Enable Feature Opt-In](/oct-22-2022-feature-opt-in-2.png)

From here, you can customize the heading, colours, logo, and descriptive text of the Feature Opt-In widget. Under the customization settings, there is a preview of the Opt-In widget where you can preview what the widget will look like based on the settings you choose.

![Feature Opt-In](/oct-22-2022-feature-opt-in-3.png)

>  Below the customization options and styling preview is the iFrame code snippet and a tool to test the widget with different user IDs. Select the Environment you would like to test or implement in the dropdown and insert any user ID to validate what the widget will look like for a given user by clicking "Open Preview". 

Author: ```suggestion
Below the customization options and styling preview is the iFrame code snippet and a tool to test the widget with different user IDs. Select the Environment you would like to test or implement in the dropdown and insert any user ID to validate what the widget will look like for a given user by clicking "Open Preview". 
```

>  This code snippet is initially presented in simple HTML format, but React and Angular alternatives are available via the "Frontend framework" dropdown above the code snippet.

Author: ```suggestion
This code snippet is initially presented in simple HTML format, but React and Angular alternatives are available via the "Frontend framework" dropdown above the code snippet.
```

>  The two most important parts of the iFrame code are the Environment key and user ID. Make sure that you are setting the right key for the Environment the widget will be displayed in, and ensure you always pass the relevant user ID for the user that is interacting with the widget.

Author: ```suggestion
The two most important parts of the iFrame code are the Environment key and user ID. Make sure that you are setting the right key for the Environment the widget will be displayed in, and ensure you always pass the relevant user ID for the user that is interacting with the widget.
```
